src/main/UserActivityMonitor.js

```diff
@@ -15,25 +15,19 @@ export default class UserActivityMonitor extends EventEmitter {
     super();
 
     this.isActive = true;
-    this.forceInactive = false;
     this.idleTime = 0;
-    this.lastStatusUpdate = 0;
+    this.lastSetActive = null;
     this.systemIdleTimeIntervalID = -1;
 
     this.config = {
-      internalUpdateFrequencyMs: 1 * 1000, // eslint-disable-line no-magic-numbers
+      updateFrequencyMs: 1 * 1000, // eslint-disable-line no-magic-numbers
```

Comment:
Nit: it’s annoying having mixed units. Worth converging on milliseconds?

Reply:
Ya, I agree. I guess since I'm already overriding eslint's `no-magic-numbers`, another one for converting to seconds won't hurt.